src/server/scripts/EasternKingdoms/MagistersTerrace/boss_vexallus.cpp

```diff
@@ -187,40 +187,7 @@ class boss_vexallus : public CreatureScript
         };
 };
 
-enum NpcPureEnergy
-{
-    SPELL_ENERGY_BOLT               = 46156,
-    SPELL_ENERGY_FEEDBACK           = 44335,
-    SPELL_PURE_ENERGY_PASSIVE       = 44326
-};
-
-class npc_pure_energy : public CreatureScript
-{
-    public:
-        npc_pure_energy() : CreatureScript("npc_pure_energy") { }
-
-        struct npc_pure_energyAI : public ScriptedAI
-        {
-            npc_pure_energyAI(Creature* creature) : ScriptedAI(creature)
-            {
-                me->SetDisplayId(me->GetCreatureTemplate()->Modelid2);
-            }
-
-            void JustDied(Unit* killer) override
-            {
```

Comment:
just add check `if (killer)`

Reply:
Yes, i will fix it when i back of uni today.